packages/trpc/server/routers/viewer/workflows/activateEventType.handler.ts

```diff
@@ -186,160 +186,164 @@ export const activateEventTypeHandler = async ({ ctx, input }: ActivateEventType
     }
     await removeSmsReminderFieldForEventTypes({ activeOnToRemove: activeOn, workflowId, isOrg });
   } else {
-    // activate workflow and schedule reminders for existing bookings
-
-    const bookingsForReminders = await prisma.booking.findMany({
-      where: {
-        eventTypeId: eventTypeId,
-        status: BookingStatus.ACCEPTED,
-        startTime: {
-          gte: new Date(),
+    if (
+      eventTypeWorkflow.trigger == WorkflowTriggerEvents.BEFORE_EVENT ||
+      eventTypeWorkflow.trigger == WorkflowTriggerEvents.AFTER_EVENT
+    ) {
```

Comment:
Adding this if statement is the only thing that changed. We want to schedule only workflow notifications with the BEFORE_EVENT and AFTER_EVENT trigger